source/normalize-arguments.ts

```diff
@@ -420,6 +420,8 @@ export const normalizeRequestArguments = async (options: NormalizedOptions): Pro
 				path,
 				host: ''
 			};
+
+			delete options.url;
```

Comment:
What is this? Seems like it should be in a separate commit/PR with a test.

Reply:
Just forgot to delete this when removing electron fixes, it's safe to remove this as `options.url` is deleted later anyway